### PR TITLE
Applicants feature

### PR DESCRIPTION
**Pull request by mquadrant**
This contains creation of applicants, modification on Record models and model seedings. Fixed failing test and added more tests.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -6,6 +6,10 @@ import morgan from 'morgan';
 import graphQLHTTP from 'express-graphql';
 import path from 'path';
 import mongoose from 'mongoose'; //This helps us to connect to our MongoDB database
+import adminCreate from './routes/admin/adminRoutes';
```
Comment (Austine3000) [inline src/app.ts]: Move these routes to routes/index

File: src/app.ts [hunk 2]
```diff
@@ -84,6 +88,9 @@ connection.on('error', () => {
   console.log('Error Connecting To Database');
 });
 app.use('/api', apiRouter);
+app.use('/api/user/admin', adminCreate);
```
Comment (Austine3000) [inline src/app.ts]: No need for this.

File: src/app.ts (hunk 1 above)
Comment (mquadrant) [reply src/app.ts]: other routes moved to index

File: src/app.ts (hunk 2 above)
Comment (mquadrant) [reply src/app.ts]: yes

**Pull request merged**